src/components/atoms/Menu.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC } from 'react';
```

Comment:
Following our discussion, it is OK to include the button, like in the [docs](https://material-ui.com/components/menus/)

You can see material uses menu for there own language control
![image](https://user-images.githubusercontent.com/11487010/88827336-843aba00-d1d2-11ea-9ffe-3b57e642f513.png)

You can include the `selected language text` in the button, or not - whatever you find easier (I believe not including it would be easier)